sdk/spring/azure-identity-spring/src/main/java/com/azure/spring/identity/AzureCloud.java

```diff
@@ -0,0 +1,27 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.spring.identity;
+
+
+/**
+ * Enum to define the authority host uri for all Azure Cloud environments: Azure, Azure China...
+ */
+public enum AzureCloud {
```

Comment:
To me this should be part of azure-core instead of be part of our spring module, since all azure sdk requires this kind of configuration.

would you confirm?